src/amazon/AmazonProduct.js

```diff
@@ -43,10 +43,10 @@ class AmazonProduct {
   }
 
   /* 
-    Determine if this amazon product has a known sell price and weight.
+    Determine if this amazon product has a known sell price.
   */
   _hasKnownInfo() {
```

Comment:
I think we should rename the method to `_hasKnownPrice` at this point.

Reply:
Renamed here 890fecf